src/WebJobs.Script.WebHost/Models/FunctionMetadataResponse.cs

```diff
@@ -46,6 +46,12 @@ public class FunctionMetadataResponse
         [JsonProperty(PropertyName = "href")]
         public Uri Href { get; set; }
 
+        /// <summary>
+        /// Gets or sets invoke url for the function, if one is supported (e.g. HTTP triggered functions)
+        /// </summary>
+        [JsonProperty(PropertyName = "invoke_url_template")]
```

Comment:
I know we wanted to use href for consistency with others, but given the fact that this is a URL template and not really a reference, I've updated to the above.

/cc @paulbatum

Reply:
Agree